src/static/js/modules/jquery/type-and-filter.js

```diff
@@ -94,7 +97,22 @@ function init() {
         .on( 'search', function() {
           $.fn.typeAndFilter.filterItems( $items, searchTerm, true, settings );
           resultsCount = $items.filter( ':visible' ).length;
-          $messages.trigger( 'searchResults' );
+          var message = settings.filteredMessageMultiple;
+          var state = 'success';
+          if ( resultsCount === 1 ) {
+            message = settings.filteredMessageSingular;
+          }
+          if ( resultsCount === 0 ) {
+            state = 'warning';
+          }
+
+          message = message.replace( /{{[\s]*term[\s]*}}/, searchTerm );
```

Comment:
This is the existing code, just moved around. I was actually thinking about replacing it with handlebars since we're already including it as a dependency.